ros_ign_bridge/src/parameter_bridge.cpp

```diff
@@ -35,22 +35,31 @@ enum Direction
   FROM_IGN_TO_ROS = 1,
   // Only from ROS to IGN
   FROM_ROS_TO_IGN = 2,
+  // Unspecified, only used for services
+  DIR_UNSPECIFIED = 3,
 };
 
 //////////////////////////////////////////////////
 void usage()
 {
-  std::cout << "Bridge a collection of ROS2 and Ignition Transport topics.\n\n" <<
-    "  parameter_bridge <topic@ROS2_type@Ign_type> .. " <<
-    " <topic@ROS2_type@Ign_type>\n\n" <<
-    "The first @ symbol delimits the topic name from the message types.\n" <<
+  std::cout << "Bridge a collection of ROS2 and Ignition Transport topics and services.\n\n" <<
+    "  parameter_bridge [<topic@ROS2_type@Ign_type> ..] " <<
+    " [<service@ROS2_srv_type[@Ign_req_type@Ign_rep_type]> ..]\n\n" <<
+    "Topics: The first @ symbol delimits the topic name from the message types.\n" <<
     "Following the first @ symbol is the ROS message type.\n" <<
     "The ROS message type is followed by an @, [, or ] symbol where\n" <<
     "    @  == a bidirectional bridge, \n" <<
     "    [  == a bridge from Ignition to ROS,\n" <<
     "    ]  == a bridge from ROS to Ignition.\n" <<
     "Following the direction symbol is the Ignition Transport message " <<
     "type.\n\n" <<
+    "Services: The first @ symbol delimits the service name from the types.\n" <<
+    "Following the first @ symbol is the ROS service type.\n" <<
+    "Optionally, you can include the ignition request and reply type\n" <<
+    "separated by the @ symbol.\n" <<
+    "It is only supported to expose Ignition servces as ROS services, i.e.\n"
+    "the ros service will forward request to the ignition service and then forward\n"
```

Comment:
```suggestion
    "the ROS service will forward request to the Ignition service and then forward\n"
```

Reply:
Fixed in https://github.com/ignitionrobotics/ros_ign/pull/211/commits/b2a37daabf00db7c9bfed437d7d096dc535f597a and https://github.com/ignitionrobotics/ros_ign/pull/211/commits/79611258cb788b3957d8e3406fe92b57a41685ba